src/50_Samples_%26_Templates/Product_Page.html

```diff
@@ -550,6 +557,8 @@ <h2 class="m1">Apple</h2>
       "selectedSlideForRed": 0,
       "selectedSlideForGolden": 0,
       "selectedSlideForGreen": 0,
+      "timesShowMoreClicked": 0,
```

Comment:
suggestion: rename `timesShowMoreClicked` to `moreItemsPageIndex`

Reply:
done